RELEASE.md

```diff
@@ -30,20 +30,24 @@ The GitHub token can be generated in GitHub/User Profile/Settings/Developer sett
 
 Once all configuration is in place, execute the following commands from the directory `java_bin/daff` (from JReleaser's [docs](https://jreleaser.org/guide/latest/examples/maven/maven-central.html)):
 
+```bash
+export version=$(grep "\"version\"" package.json | grep -E -o "[.0-9]+")
+```
+
 1) Verify release & deploy configuration
 
 ```bash
-mvn jreleaser:config
+mvn -Drevision=$version jreleaser:config
 ```
 
 2) Stage all artifacts to a local directory
 
 ```bash
-mvn -Ppublication
+mvn -Drevision=$version -Ppublication
 ```
 
 3) Deploy and release
 
 ```bash
-mvn jreleaser:full-release
+mvn -Drevision=$version jreleaser:full-release

```

Comment:
Perhaps these three steps could be top level make targets, like `make java_verify`, `make java_stage`, `make java_release`? Minor detail - the first two work for me now. The release fails with this error:
```
[ERROR] Failed to execute goal org.jreleaser:jreleaser-maven-plugin:1.8.0:full-release (default-cli) on project daff: Execution default-cli of goal org.jreleaser:jreleaser-maven-plugin:1.8.0:full-release failed: maven.nexus2.maven-central.stagingRepository does not exist: target/staging-deploy -> [Help 1]
```
I do have a directory called `./target/staging-deploys` at this point.

Reply:
My bad. There was a typo in the pom file. Now it should work :) I also made some corrections to RELEASE.md.